doc/man7/evp.pod

```diff
@@ -32,7 +32,8 @@ with a private key of a particular algorithm by using the functions
 described on the L<EVP_PKEY_fromdata(3)> page, or
 new keys can be generated using L<EVP_PKEY_keygen(3)>.
 EVP_PKEYs can be compared using L<EVP_PKEY_cmp(3)>, or printed using
-L<EVP_PKEY_print_private(3)>.
+L<EVP_PKEY_print_private(3)>. L<EVP_PKEY_todata(3)> can be used to convert a
+key back into raw parameter data.
```

Comment:
"raw" is an unfortunate term here...  how about "back into an L<OSSL_PARAM(3)> array"?